feature-detects/forcetouch.js

```diff
@@ -0,0 +1,36 @@
+/*!
+{
+  "name": "Force Touch Events",
+  "property": "forcetouch",
+  "authors": ["Kraig Walker"],
+  "notes": [{
+    "name": "Responding to Force Touch Events from JavaScript",
+    "href": "https://developer.apple.com/library/prerelease/mac/documentation/AppleApplications/Conceptual/SafariJSProgTopics/Articles/RespondingtoForceTouchEventsfromJavaScript.html"
+  }]
+}
+!*/
+/* DOC
+Tests whether the browser supports the detection of Force Touch Events.
+Force Touch Events allow custom behaviours and interactions to take place based on the given pressure or change in pressure from a compatible trackpad.
+
+Force Touch events are available in OS X 10.11 and later on devices equipped with Force Touch trackpads.
+*/
+define(['Modernizr', 'hasEvent', 'prefixed'], function(Modernizr, hasEvent, prefixed) {
+  Modernizr.addTest('forcetouch', function() {
+		// github.com/Modernizr/Modernizr/issues/1613
+    // Test if the browser supports the force touch event progression (see notes link)
+    if (!hasEvent(prefixed('mouseforcewillbegin', window, false), window) &&
```

Comment:
keying off of one of these events is fine. Its extremely unlikely to have a situation where `mouseforcewillbegin` exists but the others do not. Removign the others saves a fair amount of bits